EIPS/eip-3009.md

```diff
@@ -0,0 +1,317 @@
+---
+eip: 3009
+title: `transferWithAuthorization` - Gas-Abstracted ERC20 transactions with EIP-712
+author: Peter Jihoon Kim (@petejkim), Kevin Britz (@kbrizzle), David Knott (@DavidLKnott)
+discussions-to: https://github.com/ethereum/EIPs/issues/3010
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-28
+requires: 20, 712
+---
+
+## Simple Summary
+
+A function called `transferWithAuthorization` to enable gas-abstracted and atomic interactions with [ERC-20](https://eips.ethereum.org/EIPS/eip-20) token contracts via signatures conforming to the [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md) typed message signing specification.
```

Comment:
Links to other EIPs should be in the relative format, like this:

```suggestion
A function called `transferWithAuthorization` to enable gas-abstracted and atomic interactions with [ERC-20](./eip-20) token contracts via signatures conforming to the [EIP-712](./eip-712.md) typed message signing specification.
```

Reply:
Fixed, thank you.